src/test/resources/features/refapp-2.x/03-trackPatient/admitToInpatient.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Admit to Inpatient
+
+Background:
+    	Given a user clicks on find patient record from home page
+    	When a user selects a patient from the patient list
```

Comment:
Would you allow Given syntax to only be in background stage , As its practically good to have Given scenario in background

Reply:
Thanks @sherrif10. Let me follow up.

Reply:
@gracebish this is good. I was going to duplicate your work unknowingly, thank God I thought about looking into this ticket first. Could we have a change of the ticket title from `Create A feature file for Admit Patient.Feature` to `Admit Patient Test workflow`. @sherrif10  and @kdaud  what are your thoughts on this.